opm/polymer/fullyimplicit/PolymerPropsAd.cpp

```diff
@@ -192,6 +192,34 @@ namespace Opm {
 
 
 
+    ADB PolymerPropsAd::effectiveInvPolymerVisc(const ADB& c, const double* visc) const
+    {
+        const int nc = c.size();
+        V inv_mu_p_eff(nc);
+        V dinv_mu_p_eff(nc);
+        for (int i = 0; i < nc; ++i) {
+            double im = 0;
+            double dim = 0;
+            // TODO: the usage of visc can be likely wrong, while more investigation will be requried.
```

Comment:
I agree, the current usage is buggy and the interface is bad. Let's fix that in a separate PR.